test/routes/article.test.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  namedNode,
+} from '@rdfjs/data-model';
+import createHttpError from 'http-errors';
+import { OK } from 'http-status-codes';
+import { Response } from 'koa';
+import { DatasetCore, NamedNode } from 'rdf-js';
+import InMemoryArticles from '../../src/adaptors/in-memory-articles';
+import Articles from '../../src/articles';
+import article from '../../src/routes/article';
+import createContext from '../context';
+import createArticle from '../create-article';
+import runMiddleware, { NextMiddleware } from '../middleware';
+import { WithDataset } from '../../src/middleware/dataset';
+import ArticleNotFound from '../../src/errors/article-not-found';
+
+const inMemoryArticles = new InMemoryArticles();
```

Comment:
This will be shared between test cases, so could lead to unexpected results. Each test should have its own.